src/providers/deepgram.py

```
@@ -138,6 +143,81 @@ def build_listen_provider_block(
 )
 
 class DeepgramProvider(AIProviderInterface):
+    _FALLBACK_END_MARKERS = [
+        marker
+        for marker in DEFAULT_HANGUP_MARKERS["end_call"]
+        if marker not in {"thanks", "thank you", "no thanks", "no thank you"}
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)